src/design-system/assets-table/assets-table.stories.tsx

```diff
@@ -188,7 +208,7 @@ export const Controls: Controls = ({
               priceChange={priceChange}
               priceTrend={priceTrend}
             />
-            <TokenAmount amount={tokenAmount} fiatPrice={fiatPrice} />
+            <TokenAmount amount={tokenAmount} details={fiatPrice} />
```

Comment:
I do really like this change. It makes more sense and provides flexibility then hardcoded fiatPrice which in cases like we have with pending coins now would be semantically incorrect.